### DISCUSSION TITLE
2021/03/21/brpc

### DISCUSSION THREAD

**Discussion by giscus** [wanghenshui/wanghenshui.github.io#116] [Q&A]
# 2021/03/21/brpc

https://wanghenshui.github.io/2021/03/21/brpc.html?utterances=52b2c92fa75e84a284769f91IPcNdIevZcW39ZA%2BbHYG7vPIQy0KF%2BMD%2Bn7D7M6FUtnK%2F4CpQvxH6CNAKTV1TbuY3l%2BGkjbAGtBtqcSEfzF%2BlB2TsUPIkfu8ZXzc6CyxCbbMAU5WXdVKnncF51M%3D

**Comment by wanghenshui** [comment]
https://github.com/tsuna/contextswitch
